### PR TITLE
Better loading experience

### PR DESCRIPTION
**Pull request by jrjohnson**
Improved Ilios Loading Animation by replacing our fontawesome spinner with something ilios specific and more reliable that doesn't depend on our vendor.css file loading completely
before animation shows up.

This is a bit hard to experience on a fast device, but the slower your internet connection and browser the better it will look.

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit 8a24deb575e0c7dd711a78807b00d313b87c7333

https://deploy-preview-3326--ilios-frontend.netlify.com

**Comment by saschaben** [comment]
nice

File: lib/ilios-error/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/* eslint-env node */
+'use strict';
+
+module.exports = {
+  name: 'ilios-error',
+
+  isDevelopingAddon() {
+    return true;
```
Comment (stopfstedt) [inline lib/ilios-error/index.js]: should be set to false.

Comment (stopfstedt) [inline lib/ilios-error/index.js]: should be set to `false`.

**Comment by jrjohnson** [comment]
@stopfstedt fixed and fixed tests. This is ready for review.

**Pull request merged**